src/components/BaseFields.tsx

```diff
@@ -1,18 +1,59 @@
 import React from 'react';
 import './BaseFields.css';
 import { ApiBaseField } from '../pdc-api';
+import {
+  Table,
+  TableHead,
+  ColumnHead,
+  TableBody,
+  TableRow,
+  RowCell,
+} from './Table';
+
+interface BaseFieldListTableRowProps {
+  field: ApiBaseField;
+}
+
+const BaseFieldListTableRow = ({
+  field,
+}: BaseFieldListTableRowProps) => (
+  <TableRow>
+    <RowCell>
+      <code>
```

Comment:
Oof, those long shortCodes are a nightmare (and an oxymoron!).

We could add the Toggle Wrap button, but I'm also okay with Just Wrapping Them. Unfortunately I'm bumping up against table-column-calculation weirdness and so I think we'll have to be a bit heavy-handed with it. Here's what I suggest:

```suggestion
      <code className="base-field-short-code">
```

Then, to `BaseFields.css`, add:

```css
.base-field-short-code {
  display: block;
  max-width: 50ch;
  word-wrap: break-word;
}
```

Reply:
Nice solution! The wrapped short codes still copy fine, although selecting them is more difficult. I'm kinda okay with this subtle discouragement to using such long base fields, because that length seems like a bit of a red flag; the longer the "short" code for a base field is, the more likely we want to review it.